src/widgets/VariantInternationalComparisonChart.tsx

```diff
@@ -109,6 +111,8 @@ export const VariantInternationalComparisonChart = ({
       isFixed: false,
     }))
   );
+  const exploreUrl = useExploreUrl();
```

Comment:
The `...Chart` component should be (more or less) pure and are not supposed to fetch information through `useExploreUrl()`. Instead, it should only use information that is passed through the props. This is causing the current errors.